src/pages/new/index.tsx

```diff
@@ -8,8 +8,15 @@ import Markdown from 'react-markdown'
 import {track} from 'utils/analytics'
 import Card, {CardResource} from 'components/pages/home/card'
 import Jumbotron from 'components/pages/home/jumbotron'
+import {useTheme} from 'next-themes'
+
+const WhatsNewPage: FunctionComponent<WhatsNewResource> = ({resource}) => {
+  const [mounted, setMounted] = React.useState(false)
+  const {theme} = useTheme()
+
+  React.useEffect(() => setMounted(true), [])
+  if (!mounted) return null
```

Comment:
Was getting a hydration issue where the opposite background was being set on the first load. This ensures that the theme is properly set before rendering card backgrounds